app/views/admin/statistics_announcements/_filter_options.html.erb

```diff
@@ -0,0 +1,80 @@
+<%
+  anchor ||= anchor || ""
+  raise "filter action required" unless defined?(filter_action)
+%>
+
+<div class="app-view-filter govuk-!-margin-right-5 govuk-!-padding-5">
+  <%= form_with url: filter_action + anchor, method: :get do |_form| %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Filter by",
+      margin_bottom: 4,
+    } %>
+
+    <%= render "govuk_publishing_components/components/input", {
+      label: {
+        text: "Title or slug",
+        bold: true,
+      },
+      name: "title",
+      id: "title_filter",
+      value: @filter.options[:title],
+    } %>
+
+    <%= render "components/select-with-search", {
+      label: "Organisation",
+      id: "organisation_filter",
+      name: "organisation_id",
+      heading_size: "s",
+      grouped_options: admin_organisation_filter_options(@filter.options[:organisation_id]),
+    } %>
+
+    <%= render "govuk_publishing_components/components/select", {
+      id: "release_date_filter",
+      label: "Release date",
+      name: "dates",
+      heading_size: "s",
```

Comment:
this should probably have

```
full_width: true
```

so it is the same width as the other selects

Reply:
Yup, good spot. ✅